packages/core/lib/env.js

```diff
@@ -0,0 +1,26 @@
+const isPlainObject = require('is-plain-object');
+
+exports.environmentalize = (_config) => {
+  const _env = {};
+  const env = global._env || process.env;
```

Comment:
I would pass this as an argument as the caller knows better what actually should be used.

Reply:
I disagree - we only have a single runtime mixin container - and we would have the very same code in there, i.e. at the call site. Additionally, I think it is quite neat to have as much of the code we need to implement this feature at one place instead of having parts of it all over the place, i.e. in two more files.